modules/va_forms/app/workers/va_forms/form_builder.rb

```diff
@@ -84,15 +84,22 @@ def get_sha256(content)
       end
     end
 
-    def update_sha256(form)
+    def content(url)
       time_initial = Time.zone.now
```

Comment:
I think we might be able to gank these dirty loggers. [These spans show up in DataDog.](https://vagov.ddog-gov.com/apm/traces?query=%40_top_level%3A1%20env%3Aeks-prod%20service%3Avets-api-sidekiq%20operation_name%3Asidekiq.job%20resource_name%3A%28%22VAForms%3A%3AFormBuilder%22%29&cols=core_service%2Ccore_resource_name%2Clog_duration%2Clog_http.method%2Clog_http.status_code&graphType=flamegraph&historicalData=true&messageDisplay=inline&query_translation_version=v0&shouldShowLegend=true&sort=time&spanType=service-entry&spanViewType=logs&start=1686236811828&end=1687446411828&paused=false)

Reply:
I'm not familiar with your hip vernacular ;) Do you mean we could remove the logging entirely since the request out is automatically tracked by DataDog?

Reply:
> Could also track this as a DataDog duration event, which would give us fancy graphs and alerts from the APM instead of hard-coded, if that sound useful. Not necessary, just pointing it out :)

>Edit: the above may be moot since DataDog seems to already tracking the request out like a good little APM.

You got it. This is exactly what I was thinking.